src/transformStateWithClones.js

```diff
@@ -7,7 +7,47 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopy = {};
+
```

Comment:
You don't need to create an empty object and then use Object.assign. Instead, you can directly create a shallow copy of the initial state using the spread operator (...).
  let stateCopy = { ...state };